projects/1.16/assets/biomancy/biomancy/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+  "tooltip.biomancy.fleshborn_pressure_plate": "潜行单击以切换模式\n只有拥有者才能激活。",
+  "tooltip.biomancy.fleshborn_chest": "破坏后仍会保存内部的物品。",
+  "tooltip.biomancy.gulge": "可以储存32k相同物品，在被采集时保存其中物品。",
+  "tooltip.biomancy.chewer": "可以将把物品咀嚼成小块。\n需要提供营养来进行咀嚼。",
+  "tooltip.biomancy.digester": "消耗水将有机物变成营养物质的生物机器。\n可以从附着的容器中提取有机物。",
+  "tooltip.biomancy.solidifier": "一种可以将流体凝结成固体的分泌腺。\n可以从附着的容器或机器中提取流体。",
+  "tooltip.biomancy.decomposer": "一种可以分解物品的生物机器。\n需要提供营养来进行分解。",
+  "tooltip.biomancy.evolution_pool": "充满诱变胆汁的血肉池，用于制作有机物品。",
+  "tooltip.biomancy.boomling_hive_gun": "[WIP]与药水在工作台合成。使用营养物质重新装弹。",
+
+  "tooltip.biomancy.mining_bonus": "当挖掘 %1$s：",
+  "tooltip.biomancy.item_is_inert": "血肉现在不太活泼",
+  "tooltip.biomancy.item_is_awake": "血肉已苏醒",
+  "tooltip.biomancy.item_is_dormant": "血肉正休眠",
+  "tooltip.biomancy.item_is_excited": "血肉很兴奋",
+  "tooltip.biomancy.item_is_exalted": "血肉正亢奋",
+  "tooltip.biomancy.inert": "惰性",
+  "tooltip.biomancy.dormant": "休眠",
+  "tooltip.biomancy.awake": "苏醒",
+  "tooltip.biomancy.excited": "兴奋",
+  "tooltip.biomancy.exalted": "亢奋",
+  "tooltip.biomancy.hit_count": "命中次数：%1$sx",
+  "tooltip.biomancy.press_button_to": "按 %1$s 来 %2$s",
+  "tooltip.biomancy.action_activate": "启用",
+  "tooltip.biomancy.action_deactivate": "停用",
+  "tooltip.biomancy.action_reload": "装弹",
+  "tooltip.biomancy.action_cycle": "循环",
+  "tooltip.biomancy.action_self_inject": "对自己使用",
+  "tooltip.biomancy.contains": "包含：%1$s",
+  "tooltip.biomancy.contains_dna": "包含：%1$s 的遗传物质",
+  "tooltip.biomancy.contains_nothing": "包含：无",
+  "tooltip.biomancy.empty": "空",
+  "tooltip.biomancy.owner": "拥有者：%1$s",
+  "tooltip.biomancy.mutagen": "诱变剂",
+  "_tooltip.biomancy.slots": "Slots",
+  "tooltip.biomancy.byproduct": "副产物",
+  "tooltip.biomancy.chance": "几率",
+  "tooltip.biomancy.fire_rate": "射速",
+  "tooltip.biomancy.accuracy": "准确度",
+  "tooltip.biomancy.ammo": "弹药",
+  "tooltip.biomancy.reload_time": "装弹时间",
+  "tooltip.biomancy.projectile_damage": "弹药伤害",
+  "tooltip.biomancy.riding_bonus": "骑行时：",
+
+  "enum.biomancy.mode.none": "无",
+  "enum.biomancy.mode.devour": "吞噬模式",
+  "enum.biomancy.mode.replenish": "补充模式",
+
+  "msg.biomancy.already_full": "已经装满",
+  "msg.biomancy.not_sleepy": "你不再感觉到困了……",
+  "msg.biomancy.not_allowed": "不允许",
+  "msg.biomancy.failed_to_spawn": "创造 %1$s 失败",
+  "msg.biomancy.set_golem_command": "%1$s 现在要执行 %2$s 的命令",
+
+  "death.attack.biomancy.blight_thorn": "%1$s 被 %2$s 咬死",
+  "death.attack.biomancy.blight_thorn.item": "%1$s 被 %2$s 用 %3$s 咬死",
+  "death.attack.biomancy.ravenous_hunger": "%1$s 不能满足无尽的饥饿，最后吃了自己",
+  "death.attack.biomancy.disease": "%1$s 感染了疾病并最终死亡",
+  "death.attack.biomancy.disease.player": "%1$s 通过 %2$s 感染了疾病并最终死亡",
+  "death.attack.biomancy.tooth_projectile": "%1$s 被 %2$s 强行植入牙齿",
+  "death.attack.biomancy.tooth_projectile.item": "%1$s 被 %2$s 用 %3$s 强行植入牙齿",
+
+  "advancements.biomancy.root.title": "血肉重铸",
+  "advancements.biomancy.root.description": "血肉非常有用……",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "advancements.biomancy.root.description": "生肉非常有用……",

```